samples/analyze/beta_snippets.py

```diff
@@ -743,6 +746,83 @@ def stream_generator():
     # [END video_streaming_automl_object_tracking_beta]
 
 
+def streaming_automl_action_recognition(path, project_id, model_id):
+    # [START video_streaming_automl_action_recognition_beta]
+    import io
+
+    from google.cloud import videointelligence_v1p3beta1 as videointelligence
+
+    # path = 'path_to_file'
+    # project_id = 'project_id'
+    # model_id = 'automl_action_recognition_model_id'
+
+    client = videointelligence.StreamingVideoIntelligenceServiceClient()
+
+    model_path = "projects/{}/locations/us-central1/models/{}".format(
+        project_id, model_id
+    )
+
+    automl_config = videointelligence.StreamingAutomlActionRecognitionConfig(
+        model_name=model_path
+    )
+
+    video_config = videointelligence.StreamingVideoConfig(
+        feature=videointelligence.StreamingFeature.STREAMING_AUTOML_ACTION_RECOGNITION,
+        automl_action_recognition_config=automl_config,
+    )
+
+    # config_request should be the first in the stream of requests.
+    config_request = videointelligence.StreamingAnnotateVideoRequest(
+        video_config=video_config
+    )
+
+    # Set the chunk size to 5MB (recommended less than 10MB).
+    chunk_size = 5 * 1024 * 1024
+
+    # Load file content.
+    # Note: Input videos must have supported video codecs. See
+    # https://cloud.google.com/video-intelligence/docs/streaming/streaming#supported_video_codecs
+    # for more details.
+    stream = []
+    with io.open(path, "rb") as video_file:
```

Comment:
Can the `video_file` be processed inside `stream_generator` (removing this with+while loop)? Filling the list with chunks and then creating a generator feels a bit odd to me.

Reply:
Done.